src/xregexp.js

```diff
@@ -916,18 +928,25 @@ XRegExp.globalize = (regex) => copyRegex(regex, {addG: true});
  * // With an options object
  * XRegExp.install({
  *   // Enables support for astral code points in Unicode addons (implicitly sets flag A)
- *   astral: true
+ *   astral: true,
+ *
+ *   // Enables named capture groups
```

Comment:
This seems misleading, since named capture groups are always enabled. What about "``Moves named backreferences to the `groups` property of match arrays``", "``Use a `groups` object to store named backreferences``", or something similar?

Reply:
Good catch! In https://github.com/slevithan/xregexp/pull/210/commits/24a29d0b2e4a7fa5aad0da028cce33521fe9863a ,I decided to go with:

    Adds named capture groups to the `groups` property of match arrays.

since "capture groups" seemed a little more accurate than "backreferences", and it matches the wording in the TC39 proposal.